index.html

```diff
@@ -102,9 +120,41 @@ <h1 id="aemp-title">
       </div>
     </script>
 
+    <script id="aemp-rentstrike-infowindow-template" type="x-tmpl-mustache">
+      <div class="aemp-infowindow">
+        <a class="aemp-infowindow-close" href="#close" onClick="closeInfo()">×</a>
+        <div>
```

Comment:
@timhitchins I think it would be good to add a title to both infowindow templates to differentiate them.
e.g. the rent strike template could have a title text of "Rent Strike" and the protections template could have a title of "Emergency Tenant Protections". Currently when clicking on a rent strike marker it's not clear what it represents and I think this would help.

Reply:
@clhenrick  All great catches and suggestions.  I will dive into these tonight.